backend/eventd/eventd.go

```diff
@@ -70,6 +79,9 @@ func New(c Config, opts ...Option) (*Eventd, error) {
 			return nil, err
 		}
 	}
+
+	_ = prometheus.Register(eventsProcessed)
```

Comment:
Should this error be returned by New if non-nil?

Reply:
I don't believe so, I wouldn't want exposing telemetry to throw a wrench into the works. See git history for context.